src/features/references/editor/__tests__/ReferencesTableView.test.tsx

```diff
@@ -117,24 +118,21 @@ describe('ReferencesTableView component', () => {
   it(`should not emit ${'refstudio://references/remove' as RefStudioEventName} with NO selections`, async () => {
     const { user } = setupWithJotaiProvider(<ReferencesTableView />, store);
 
-    const actionsMenu = screen.getByTestId('actions-menu');
-    await user.hover(actionsMenu);
-    const removeBtn = within(actionsMenu).getByText('Remove');
+    const removeBtn = screen.getByText('Remove');
     await user.click(removeBtn);
 
+    expect(removeBtn).toBeDisabled();
```

Comment:
Assuming you remove `disabled={disabled}` in the button component this would also be removed.